Documentation/Blazorise.Docs/Pages/Docs/Components/Links/LinkPage.razor

```diff
@@ -121,6 +121,9 @@
     <DocsAttributesItem Name="Match" Type="Match" Default="Prefix">
         URL matching behavior for a link.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="CustomMatch" Type="Func<string, bool>" Default="null">
+        A callback function that is used to compare current uri with the user defined uri. Must enable <Code>Match="Match.Custom"</Code> to be used.
```

Comment:
Why do we need to enable `Match.Custom`? Just by providing the function, should be enough on itself?
We're getting on a place where `CustomMatch` can be defined and not work at all, because `Match.Custom` was not defined, and I'm not sure there's a benefit to that, it's just needlessly more verbose for the user?

Reply:
We could change it but that is a breaking change then. Maybe for 2.0.

Reply:
Just open a 2.0 task then and keep it like this for now I guess.

Reply:
done